components/AnimatedMasoniteLogo/style.module.scss

```diff
@@ -1,6 +1,5 @@
-.centered {
-  position: fixed;
-  left: 50vw;
-  top: 50vh;
-  transform: translate3d(-50%, -50%, 0);
+@import "styles/index";
+
+.anime-bg-black {
```

Comment:
Let's update this to `'.animated-masonite-logo--dark'` and let's add a new class `.animated-masonite-logo` with `display: inline-block`. If you look in Storybook for the logo, you'll see that it looks a little odd because it is display block.